content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
 1. Select or create a project on the [GCP Console][gcp-console].
 1. Make sure that billing is enabled for your project. See the guide to
   [modifying a project's billing settings][billing-guide].
-1. Install the [Cloud SDK][cloud-sdk].
+1. Use Cloud Shell to grant your team access to Kubeflow by granting them **IAP-secured Web App User** roles:
+
+      ```
+      gcloud projects add-iam-policy-binding <PROJECT>
+      --member user:<EMAIL> --role IAP-secured Web App User
+      ```
+    For more details, see the [granting access to GCP resources guide][granting-changing-revoking-access].
 
 Notes:
 
 * As you work through this tutorial, your project uses billable components of
   GCP. To minimise costs, follow the instructions to
   [clean up your GCP resources](#cleanup) when you've finished with them.
-* This guide assumes you want to manage your GCP environment on your own server
-  rather than in the [Cloud Shell][cloud-shell] environment. If you choose to
-  use the Cloud Shell, some of the components are pre-installed in your shell.
-
-### (Optional) Install kubectl
-
-`kubectl` is the command-line tool for Kubernetes. `kubectl` is useful if you 
-want to interact with your Kubeflow cluster locally. If you decide not to use
-`kubectl`, you can skip the steps later in the tutorial that use the command.
-
-Run the following Cloud SDK command to install `kubectl`:
-
-```
-gcloud components install kubectl
-```
+* This guide assumes you want to manage your GCP environment in the [Cloud Shell][cloud-shell] environment rather than your own server. If you choose to use a local server instead, some of the components (such as Cloud SDK) must be manually installed.
```

Comment:
Suggestion: We are recommending that they use Cloud Shell to save users the steps of installing Cloud SDK and kubectl. Let's rephrase this so users have a better understanding of the reason for the recommendation. How about something like this?

* This guide uses [Cloud Shell][cloud-shell] to manage your GCP environment, to save you the steps of installing [Cloud SDK][gcloud-install] and [kubectl][kubectl-install]. 

[kubectl-install]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
[gcloud-install]: https://cloud.google.com/sdk/

Reply:
Agreed and appreciated! I have updated the text accordingly.